src/components/CanvasWrapper.tsx

```diff
@@ -752,24 +751,22 @@ export default function CanvasWrapper(inputProps: CanvasWrapperProps): ReactElem
         </TooltipWithSubtitle>
 
         {/* Annotation mode toggle */}
-        {INTERNAL_BUILD && (
-          <TooltipWithSubtitle
-            title={props.annotationState.visible ? "Hide annotations" : "Show annotations"}
-            subtitleList={annotationTooltipContents}
-            placement="right"
-            trigger={["hover", "focus"]}
+        <TooltipWithSubtitle
+          title={props.annotationState.visible ? "Hide annotations" : "Show annotations"}
+          subtitleList={annotationTooltipContents}
+          placement="right"
+          trigger={["hover", "focus"]}
+        >
+          <IconButton
+            type={props.annotationState.visible ? "primary" : "link"}
+            onClick={() => {
+              props.annotationState.setVisibility(!props.annotationState.visible);
+            }}
           >
-            <IconButton
-              type={props.annotationState.visible ? "primary" : "link"}
-              onClick={() => {
-                props.annotationState.setVisibility(!props.annotationState.visible);
-              }}
-            >
-              {props.annotationState.visible ? <TagSlashIconSVG /> : <TagIconSVG />}
-              <VisuallyHidden>{props.annotationState.visible ? "Hide annotations" : "Show annotations"}</VisuallyHidden>
-            </IconButton>
-          </TooltipWithSubtitle>
-        )}
```

Comment:
Just removed the `INTERNAL_BUILD` conditional here. I forgot to do this in #676 :X